### DISCUSSION TITLE
NBA TV Philippines

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#675] [Q&A]
**Channel Name:** NBA TV Philippines
**Channel ID:** NBATVPhilippines.ph
**Country:** Philippines
**Language:** English
**External Link (optional):** xxx
**Notes (optional):** Philippine version of NBA TV.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19673
